clib/helpstr.c

```diff
@@ -26,13 +27,92 @@
 /*  any type of equipment. An m is for Mark III, and a v is for   */
 /*  VLBA.                                                         */
 
+#include <dirent.h>
+#include <limits.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
+#include <sys/types.h>
+#include <unistd.h>
+
 #include "../include/params.h"
 
-#define MAX_STRING  256
+#define MAX_STRING 256
+
+// fileexists check if file exists and is readable
+static int fileexists(const char *filename) {
+	FILE *file;
+	if ((file = fopen(filename, "r"))) {
+		fclose(file);
+		return 1;
+	}
+	return 0;
```

Comment:
Maybe:

````
-       if ((file = fopen(filename, "r"))) {
-               fclose(file);
-               return 1;
-       }
-       return 0;
+       if (access(filename, R_OK))
+           return 0;
+
+       return 1;
````

Reply:
Ooh nice I didn't know about that one. We should be able to remove this routine altogether.